sync2jira/downstream_issue.py

```diff
@@ -152,6 +152,68 @@ def execute_snowflake_query(issue):
     return results
 
 
+def _get_field_id_by_name(client, field_name):
+    """
+    Convert a human-readable custom field name to its JIRA field ID.
+
+    :param jira.client.JIRA client: JIRA client
+    :param str field_name: Human-readable field name (e.g., "Story Points", "Epic Link")
+    :returns: Field ID (e.g., "customfield_12310243") or None if not found
+    :rtype: Optional[str]
+    """
+    # Cache field mappings per client to avoid repeated API calls
+    cache_key = id(client)
+    if cache_key not in jira_cache:
+        jira_cache[cache_key] = {}
```

Comment:
I'm hoping that you're going to just delete this code; however, if you decide to keep it, then there's no point to initializing `jira_cache[cache_key]` to `{}`:  instead, you want the code which calls `client.fields()`...because you _know_ that the lookup in the next block is going to fail.